src/main/docs/guide/dbc/jdbc/jdbcQuickStart.adoc

```diff
@@ -71,11 +71,11 @@ NOTE: The `schema-generate` setting is only useful for demos and testing trivial
 
 To retrieve objects from the database you need to define a class annotated with ann:data.annotation.MappedEntity[]. Note that this is a meta annotation and in fact if you prefer you can use JPA annotations (only a subset are supported, more on that later). If you wish to use JPA annotations include the following `compileOnly` scoped dependency:
 
-dependency:jakarta.persistence:jakarta.persistence-api[version="3.0.0", scope="compileOnly"]
+dependency:jakarta.persistence:jakarta.persistence-api[scope="compileOnly"]
 
 To use JPA annotations in the `javax.persistence` package use:
 
-dependency:jakarta.persistence:jakarta.persistence-api[version="2.2.3", scope="compileOnly"]
+dependency:jakarta.persistence:jakarta.persistence-api[version="3.0.0", scope="compileOnly"]
```

Comment:
```suggestion
dependency:javax.persistence:javax.persistence-api[scope="compileOnly"]
```

Reply:
@radovanradic @dstepanov I assume this was a bug we were not mentioning the correct maven coordinates. 

[Making the dependencies managed](https://github.com/micronaut-projects/micronaut-data/pull/2428), should allow us to remove the version number.